packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/local/auth_token_shared_dara_source.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021, Prime Holding JSC
+// https://www.primeholding.com
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import 'package:shared_preferences/shared_preferences.dart';
+
+import 'auth_token_data_source.dart';
+
+/// Concrete implementation of AuthTokenDataSource using SharedPreferences.
+/// Suitable for mobile and web.
+/// Persist and get auth information to make it  available trough the app.
+class AuthTokenSharedDataSource implements AuthTokenDataSource {
+
+  /// Get stored access token
+  @override
+  Future<String?> getToken() async {
+    final prefs = await SharedPreferences.getInstance();
```

Comment:
Same comment as above. Always p Po refer the explicit dependencies over the implicit one